tv_apps/tv-epg/manifest.webapp

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "EPG",
+  "description": "Electronic program guide",
+  "launch_path": "/index.html",
+  "type": "certified",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "locales": {
+    "en-US": {
+      "name": "Program Guide",
+      "description": "Electronic program guide"
+    }
+  },
+  "default_locale": "en-US",
+  "icons": {
+    "354": "/style/icons/ic_default_app.png"
+  },
+  "orientation": "landscape",
+  "permissions": {
+    "tv": {},
+    "webapps-manage": {},
```

Comment:
Why this permission?

I don't think we need them now. Please review `webapps-manage` and `systemXHR` permission. We should add them once we add the feature who requires them.